packages/blockchain-link-types/package.json

```diff
@@ -22,5 +22,8 @@
         "socks-proxy-agent": "6.1.1",
         "tsx": "^3.12.7",
         "typescript": "4.9.5"
+    },
+    "dependencies": {
+        "@solana/web3.js": "^1.78.5"
```

Comment:
Can we please keep the dependency out of `blockchain-link-types` package? It seems to be unused anyway.

Reply:
i guess it is used now after rebase, lets revisit this later

Reply:
IMO it should be only in `devDependencies` which I already introduced in `feat/solana` @mroz22 @memoriesadrift